File: generated_code_examples/c_sharp/classification/svm.cs
using static System.Math;
namespace ML {
    public static class Model {
        public static double[] Score(double[] input) {
            double var0;
            var0 = Exp(-0.06389634699048878 * (Pow(5.1 - input[0], 2.0) + Pow(2.5 - input[1], 2.0) + Pow(3.0 - input[2], 2.0) + Pow(1.1 - input[3], 2.0)));
            double var1;
            var1 = Exp(-0.06389634699048878 * (Pow(4.9 - input[0], 2.0) + Pow(2.4 - input[1], 2.0) + Pow(3.3 - input[2], 2.0) + Pow(1.0 - input[3], 2.0)));
            double var2;
            var2 = Exp(-0.06389634699048878 * (Pow(6.3 - input[0], 2.0) + Pow(2.5 - input[1], 2.0) + Pow(4.9 - input[2], 2.0) + Pow(1.5 - input[3], 2.0)));
            double var3;
            var3 = Exp(-0.06389634699048878 * (Pow(5.4 - input[0], 2.0) + Pow(3.0 - input[1], 2.0) + Pow(4.5 - input[2], 2.0) + Pow(1.5 - input[3], 2.0)));
            double var4;
            var4 = Exp(-0.06389634699048878 * (Pow(6.2 - input[0], 2.0) + Pow(2.2 - input[1], 2.0) + Pow(4.5 - input[2], 2.0) + Pow(1.5 - input[3], 2.0)));
            double var5;
            var5 = Exp(-0.06389634699048878 * (Pow(5.6 - input[0], 2.0) + Pow(2.9 - input[1], 2.0) + Pow(3.6 - input[2], 2.0) + Pow(1.3 - input[3], 2.0)));
            double var6;
            var6 = Exp(-0.06389634699048878 * (Pow(6.7 - input[0], 2.0) + Pow(3.0 - input[1], 2.0) + Pow(5.0 - input[2], 2.0) + Pow(1.7 - input[3], 2.0)));
            double var7;
            var7 = Exp(-0.06389634699048878 * (Pow(5.0 - input[0], 2.0) + Pow(2.3 - input[1], 2.0) + Pow(3.3 - input[2], 2.0) + Pow(1.0 - input[3], 2.0)));
            double var8;
            var8 = Exp(-0.06389634699048878 * (Pow(6.0 - input[0], 2.0) + Pow(2.7 - input[1], 2.0) + Pow(5.1 - input[2], 2.0) + Pow(1.6 - input[3], 2.0)));
            double var9;
            var9 = Exp(-0.06389634699048878 * (Pow(5.9 - input[0], 2.0) + Pow(3.2 - input[1], 2.0) + Pow(4.8 - input[2], 2.0) + Pow(1.8 - input[3], 2.0)));
            double var10;
            var10 = Exp(-0.06389634699048878 * (Pow(5.7 - input[0], 2.0) + Pow(2.6 - input[1], 2.0) + Pow(3.5 - input[2], 2.0) + Pow(1.0 - input[3], 2.0)));
            double var11;
            var11 = Exp(-0.06389634699048878 * (Pow(5.0 - input[0], 2.0) + Pow(3.0 - input[1], 2.0) + Pow(1.6 - input[2], 2.0) + Pow(0.2 - input[3], 2.0)));
            double var12;
            var12 = Exp(-0.06389634699048878 * (Pow(5.4 - input[0], 2.0) + Pow(3.4 - input[1], 2.0) + Pow(1.7 - input[2], 2.0) + Pow(0.2 - input[3], 2.0)));
            double var13;
            var13 = Exp(-0.06389634699048878 * (Pow(5.7 - input[0], 2.0) + Pow(3.8 - input[1], 2.0) + Pow(1.7 - input[2], 2.0) + Pow(0.3 - input[3], 2.0)));
            double var14;
            var14 = Exp(-0.06389634699048878 * (Pow(4.8 - input[0], 2.0) + Pow(3.4 - input[1], 2.0) + Pow(1.9 - input[2], 2.0) + Pow(0.2 - input[3], 2.0)));
            double var15;
            var15 = Exp(-0.06389634699048878 * (Pow(4.5 - input[0], 2.0) + Pow(2.3 - input[1], 2.0) + Pow(1.3 - input[2], 2.0) + Pow(0.3 - input[3], 2.0)));
            double var16;
            var16 = Exp(-0.06389634699048878 * (Pow(5.7 - input[0], 2.0) + Pow(4.4 - input[1], 2.0) + Pow(1.5 - input[2], 2.0) + Pow(0.4 - input[3], 2.0)));
            double var17;
            var17 = Exp(-0.06389634699048878 * (Pow(5.1 - input[0], 2.0) + Pow(3.8 - input[1], 2.0) + Pow(1.9 - input[2], 2.0) + Pow(0.4 - input[3], 2.0)));
            double var18;
            var18 = Exp(-0.06389634699048878 * (Pow(5.1 - input[0], 2.0) + Pow(3.3 - input[1], 2.0) + Pow(1.7 - input[2], 2.0) + Pow(0.5 - input[3], 2.0)));
            double var19;
            var19 = Exp(-0.06389634699048878 * (Pow(6.2 - input[0], 2.0) + Pow(2.8 - input[1], 2.0) + Pow(4.8 - input[2], 2.0) + Pow(1.8 - input[3], 2.0)));
            double var20;
            var20 = Exp(-0.06389634699048878 * (Pow(7.2 - input[0], 2.0) + Pow(3.0 - input[1], 2.0) + Pow(5.8 - input[2], 2.0) + Pow(1.6 - input[3], 2.0)));
            double var21;
            var21 = Exp(-0.06389634699048878 * (Pow(6.1 - input[0], 2.0) + Pow(3.0 - input[1], 2.0) + Pow(4.9 - input[2], 2.0) + Pow(1.8 - input[3], 2.0)));
            double var22;
            var22 = Exp(-0.06389634699048878 * (Pow(6.0 - input[0], 2.0) + Pow(3.0 - input[1], 2.0) + Pow(4.8 - input[2], 2.0) + Pow(1.8 - input[3], 2.0)));
            double var23;
            var23 = Exp(-0.06389634699048878 * (Pow(4.9 - input[0], 2.0) + Pow(2.5 - input[1], 2.0) + Pow(4.5 - input[2], 2.0) + Pow(1.7 - input[3], 2.0)));
            double var24;
            var24 = Exp(-0.06389634699048878 * (Pow(7.9 - input[0], 2.0) + Pow(3.8 - input[1], 2.0) + Pow(6.4 - input[2], 2.0) + Pow(2.0 - input[3], 2.0)));
            double var25;
            var25 = Exp(-0.06389634699048878 * (Pow(5.6 - input[0], 2.0) + Pow(2.8 - input[1], 2.0) + Pow(4.9 - input[2], 2.0) + Pow(2.0 - input[3], 2.0)));
            double var26;
            var26 = Exp(-0.06389634699048878 * (Pow(6.0 - input[0], 2.0) + Pow(2.2 - input[1], 2.0) + Pow(5.0 - input[2], 2.0) + Pow(1.5 - input[3], 2.0)));
            double var27;
            var27 = Exp(-0.06389634699048878 * (Pow(6.3 - input[0], 2.0) + Pow(2.8 - input[1], 2.0) + Pow(5.1 - input[2], 2.0) + Pow(1.5 - input[3], 2.0)));
            return new double[3] {0.11172510039290856 + var0 * -0.8898986041811555 + var1 * -0.8898986041811555 + var2 * -0.0 + var3 * -0.0 + var4 * -0.0 + var5 * -0.756413813553974 + var6 * -0.0 + var7 * -0.8898986041811555 + var8 * -0.0 + var9 * -0.0 + var10 * -0.8898986041811555 + var11 * 0.04218875216876044 + var12 * 0.7142250613852136 + var13 * 0.0 + var14 * 0.8898986041811555 + var15 * 0.8898986041811555 + var16 * 0.0 + var17 * 0.8898986041811555 + var18 * 0.8898986041811555, -0.04261957451303831 + var19 * -0.37953658977037247 + var20 * -0.0 + var21 * -0.0 + var22 * -0.37953658977037247 + var23 * -0.37953658977037247 + var24 * -0.26472396872040066 + var25 * -0.3745962010653211 + var26 * -0.10077618026650095 + var27 * -0.0 + var11 * 0.0 + var12 * 0.0 + var13 * 0.37953658977037247 + var14 * 0.37953658977037247 + var15 * 0.3044555865539922 + var16 * 0.05610417372785803 + var17 * 0.37953658977037247 + var18 * 0.37953658977037247, 1.8136162062461285 + var19 * -110.34516826676301 + var20 * -13.999391039896215 + var21 * -108.44329471899991 + var22 * -110.34516826676301 + var23 * -22.21095753342801 + var24 * -0.0 + var25 * -0.0 + var26 * -65.00217641452454 + var27 * -110.34516826676301 + var0 * 0.0 + var1 * 0.0 + var2 * 110.34516826676301 + var3 * 62.115561183470184 + var4 * 37.19509025661546 + var5 * 0.0 + var6 * 110.34516826676301 + var7 * 0.0 + var8 * 110.34516826676301 + var9 * 110.34516826676301 + var10 * 0.0};
        }
    }
}

File: generated_code_examples/fortran/classification/svm.f90
module Model
    implicit none
contains
    function score(input)
        implicit none
        double precision, dimension(:) :: input
        double precision :: var0
        double precision :: var1
        double precision :: var2
        double precision :: var3
        double precision :: var4
        double precision :: var5
        double precision :: var6
        double precision :: var7
        double precision :: var8
        double precision :: var9
        double precision :: var10
        double precision :: var11
        double precision :: var12
        double precision :: var13
        double precision :: var14
        double precision :: var15
        double precision :: var16
        double precision :: var17
        double precision :: var18
        double precision :: var19
        double precision :: var20
        double precision :: var21
        double precision :: var22
        double precision :: var23
        double precision :: var24
        double precision :: var25
        double precision :: var26
        double precision :: var27
        double precision, dimension(3) :: score
        var0 = EXP(-0.06389634699048878d0 * ((5.1d0 - input(1)) ** 2.0d0 + (2.5d0 - input(2)) ** 2.0d0 + (3.0d0 - input(3)) ** 2.0d0 + (1.1d0 - input(4)) ** 2.0d0))
        var1 = EXP(-0.06389634699048878d0 * ((4.9d0 - input(1)) ** 2.0d0 + (2.4d0 - input(2)) ** 2.0d0 + (3.3d0 - input(3)) ** 2.0d0 + (1.0d0 - input(4)) ** 2.0d0))
        var2 = EXP(-0.06389634699048878d0 * ((6.3d0 - input(1)) ** 2.0d0 + (2.5d0 - input(2)) ** 2.0d0 + (4.9d0 - input(3)) ** 2.0d0 + (1.5d0 - input(4)) ** 2.0d0))
        var3 = EXP(-0.06389634699048878d0 * ((5.4d0 - input(1)) ** 2.0d0 + (3.0d0 - input(2)) ** 2.0d0 + (4.5d0 - input(3)) ** 2.0d0 + (1.5d0 - input(4)) ** 2.0d0))
        var4 = EXP(-0.06389634699048878d0 * ((6.2d0 - input(1)) ** 2.0d0 + (2.2d0 - input(2)) ** 2.0d0 + (4.5d0 - input(3)) ** 2.0d0 + (1.5d0 - input(4)) ** 2.0d0))
        var5 = EXP(-0.06389634699048878d0 * ((5.6d0 - input(1)) ** 2.0d0 + (2.9d0 - input(2)) ** 2.0d0 + (3.6d0 - input(3)) ** 2.0d0 + (1.3d0 - input(4)) ** 2.0d0))
        var6 = EXP(-0.06389634699048878d0 * ((6.7d0 - input(1)) ** 2.0d0 + (3.0d0 - input(2)) ** 2.0d0 + (5.0d0 - input(3)) ** 2.0d0 + (1.7d0 - input(4)) ** 2.0d0))
        var7 = EXP(-0.06389634699048878d0 * ((5.0d0 - input(1)) ** 2.0d0 + (2.3d0 - input(2)) ** 2.0d0 + (3.3d0 - input(3)) ** 2.0d0 + (1.0d0 - input(4)) ** 2.0d0))
        var8 = EXP(-0.06389634699048878d0 * ((6.0d0 - input(1)) ** 2.0d0 + (2.7d0 - input(2)) ** 2.0d0 + (5.1d0 - input(3)) ** 2.0d0 + (1.6d0 - input(4)) ** 2.0d0))
        var9 = EXP(-0.06389634699048878d0 * ((5.9d0 - input(1)) ** 2.0d0 + (3.2d0 - input(2)) ** 2.0d0 + (4.8d0 - input(3)) ** 2.0d0 + (1.8d0 - input(4)) ** 2.0d0))
        var10 = EXP(-0.06389634699048878d0 * ((5.7d0 - input(1)) ** 2.0d0 + (2.6d0 - input(2)) ** 2.0d0 + (3.5d0 - input(3)) ** 2.0d0 + (1.0d0 - input(4)) ** 2.0d0))
        var11 = EXP(-0.06389634699048878d0 * ((5.0d0 - input(1)) ** 2.0d0 + (3.0d0 - input(2)) ** 2.0d0 + (1.6d0 - input(3)) ** 2.0d0 + (0.2d0 - input(4)) ** 2.0d0))
        var12 = EXP(-0.06389634699048878d0 * ((5.4d0 - input(1)) ** 2.0d0 + (3.4d0 - input(2)) ** 2.0d0 + (1.7d0 - input(3)) ** 2.0d0 + (0.2d0 - input(4)) ** 2.0d0))
        var13 = EXP(-0.06389634699048878d0 * ((5.7d0 - input(1)) ** 2.0d0 + (3.8d0 - input(2)) ** 2.0d0 + (1.7d0 - input(3)) ** 2.0d0 + (0.3d0 - input(4)) ** 2.0d0))
        var14 = EXP(-0.06389634699048878d0 * ((4.8d0 - input(1)) ** 2.0d0 + (3.4d0 - input(2)) ** 2.0d0 + (1.9d0 - input(3)) ** 2.0d0 + (0.2d0 - input(4)) ** 2.0d0))
        var15 = EXP(-0.06389634699048878d0 * ((4.5d0 - input(1)) ** 2.0d0 + (2.3d0 - input(2)) ** 2.0d0 + (1.3d0 - input(3)) ** 2.0d0 + (0.3d0 - input(4)) ** 2.0d0))
        var16 = EXP(-0.06389634699048878d0 * ((5.7d0 - input(1)) ** 2.0d0 + (4.4d0 - input(2)) ** 2.0d0 + (1.5d0 - input(3)) ** 2.0d0 + (0.4d0 - input(4)) ** 2.0d0))
        var17 = EXP(-0.06389634699048878d0 * ((5.1d0 - input(1)) ** 2.0d0 + (3.8d0 - input(2)) ** 2.0d0 + (1.9d0 - input(3)) ** 2.0d0 + (0.4d0 - input(4)) ** 2.0d0))
        var18 = EXP(-0.06389634699048878d0 * ((5.1d0 - input(1)) ** 2.0d0 + (3.3d0 - input(2)) ** 2.0d0 + (1.7d0 - input(3)) ** 2.0d0 + (0.5d0 - input(4)) ** 2.0d0))
        var19 = EXP(-0.06389634699048878d0 * ((6.2d0 - input(1)) ** 2.0d0 + (2.8d0 - input(2)) ** 2.0d0 + (4.8d0 - input(3)) ** 2.0d0 + (1.8d0 - input(4)) ** 2.0d0))
        var20 = EXP(-0.06389634699048878d0 * ((7.2d0 - input(1)) ** 2.0d0 + (3.0d0 - input(2)) ** 2.0d0 + (5.8d0 - input(3)) ** 2.0d0 + (1.6d0 - input(4)) ** 2.0d0))
        var21 = EXP(-0.06389634699048878d0 * ((6.1d0 - input(1)) ** 2.0d0 + (3.0d0 - input(2)) ** 2.0d0 + (4.9d0 - input(3)) ** 2.0d0 + (1.8d0 - input(4)) ** 2.0d0))
        var22 = EXP(-0.06389634699048878d0 * ((6.0d0 - input(1)) ** 2.0d0 + (3.0d0 - input(2)) ** 2.0d0 + (4.8d0 - input(3)) ** 2.0d0 + (1.8d0 - input(4)) ** 2.0d0))
        var23 = EXP(-0.06389634699048878d0 * ((4.9d0 - input(1)) ** 2.0d0 + (2.5d0 - input(2)) ** 2.0d0 + (4.5d0 - input(3)) ** 2.0d0 + (1.7d0 - input(4)) ** 2.0d0))
        var24 = EXP(-0.06389634699048878d0 * ((7.9d0 - input(1)) ** 2.0d0 + (3.8d0 - input(2)) ** 2.0d0 + (6.4d0 - input(3)) ** 2.0d0 + (2.0d0 - input(4)) ** 2.0d0))
        var25 = EXP(-0.06389634699048878d0 * ((5.6d0 - input(1)) ** 2.0d0 + (2.8d0 - input(2)) ** 2.0d0 + (4.9d0 - input(3)) ** 2.0d0 + (2.0d0 - input(4)) ** 2.0d0))
        var26 = EXP(-0.06389634699048878d0 * ((6.0d0 - input(1)) ** 2.0d0 + (2.2d0 - input(2)) ** 2.0d0 + (5.0d0 - input(3)) ** 2.0d0 + (1.5d0 - input(4)) ** 2.0d0))
        var27 = EXP(-0.06389634699048878d0 * ((6.3d0 - input(1)) ** 2.0d0 + (2.8d0 - input(2)) ** 2.0d0 + (5.1d0 - input(3)) ** 2.0d0 + (1.5d0 - input(4)) ** 2.0d0))
        score(:) = (/ 0.11172510039290856d0 + var0 * -0.8898986041811555d0 + var1 * -0.8898986041811555d0 + var2 * -0.0d0 + var3 * -0.0d0 + var4 * -0.0d0 + var5 * -0.756413813553974d0 + var6 * -0.0d0 + var7 * -0.8898986041811555d0 + var8 * -0.0d0 + var9 * -0.0d0 + var10 * -0.8898986041811555d0 + var11 * 0.04218875216876044d0 + var12 * 0.7142250613852136d0 + var13 * 0.0d0 + var14 * 0.8898986041811555d0 + var15 * 0.8898986041811555d0 + var16 * 0.0d0 + var17 * 0.8898986041811555d0 + var18 * 0.8898986041811555d0, -0.04261957451303831d0 + var19 * -0.37953658977037247d0 + var20 * -0.0d0 + var21 * -0.0d0 + var22 * -0.37953658977037247d0 + var23 * -0.37953658977037247d0 + var24 * -0.26472396872040066d0 + var25 * -0.3745962010653211d0 + var26 * -0.10077618026650095d0 + var27 * -0.0d0 + var11 * 0.0d0 + var12 * 0.0d0 + var13 * 0.37953658977037247d0 + var14 * 0.37953658977037247d0 + var15 * 0.3044555865539922d0 + var16 * 0.05610417372785803d0 + var17 * 0.37953658977037247d0 + var18 * 0.37953658977037247d0, 1.8136162062461285d0 + var19 * -110.34516826676301d0 + var20 * -13.999391039896215d0 + var21 * -108.44329471899991d0 + var22 * -110.34516826676301d0 + var23 * -22.21095753342801d0 + var24 * -0.0d0 + var25 * -0.0d0 + var26 * -65.00217641452454d0 + var27 * -110.34516826676301d0 + var0 * 0.0d0 + var1 * 0.0d0 + var2 * 110.34516826676301d0 + var3 * 62.115561183470184d0 + var4 * 37.19509025661546d0 + var5 * 0.0d0 + var6 * 110.34516826676301d0 + var7 * 0.0d0 + var8 * 110.34516826676301d0 + var9 * 110.34516826676301d0 + var10 * 0.0d0 /)
        return
    end function score
end module Model
